project-progression-policy.md

```diff
@@ -140,20 +139,19 @@ To be considered for the Sandbox Stage, the project must meet the following requ
 ### Incubation Stage
 **Definition**
 
-The Incubation stage is for projects that are interested in reaching the Graduation stage, and have identified a [growth plan](growth-plans.md) for doing so. Incubation stage projects will receive mentorship from the TAC and are expected to actively develop their community of contributors, governance, project documentation, and other variables identified in the growth plan that factor into broad success and adoption.
+The Incubation stage is for projects that have identified a [growth plan](growth-plans.md). Incubation stage projects will receive mentorship from the TAC and are expected to actively develop their community of contributors, governance, project documentation, and other variables identified in the growth plan that factor into broad success and adoption.
 
 In order to support their active development, projects in the Incubation stage have a higher level of access to Consortium resources as provided by the Governing Board of the Consortium. A project's progress toward its growth plan goals will be reviewed on a yearly basis, and the TAC may move a project to the Sandbox stage if progress on the plan drops off or stalls.
 
 **Examples**
 
-1.	Projects that are on their way or very likely to become Graduation projects.
-2.	Projects that have developed new growth targets or other community metrics for success.
-3.	Projects that are looking to create a lifecycle plan (maintainership succession, contributor programs, version planning, etc.)
-4.	Projects that need more active support from the Consortium or TAC mentorship in order to reach their goals.
+1.	Projects that have developed new growth targets or other community metrics for success.
+2.	Projects that are looking to create a lifecycle plan (maintainership succession, contributor programs, version planning, etc.)
+3.	Projects that need more active support from the Consortium or TAC mentorship in order to reach their goals.
 
 **Expectations**
 
-Projects in the Incubation stage are generally expected to move out of the Incubation stage within two years. Depending on their growth plans, projects may cycle through Sandbox, Incubation, or Graduation stage as needed.
+Projects in the Incubation stage are generally expected to move out of the Incubation stage within two years. Depending on their growth plans, projects may cycle through Sandbox or Incubation stage as needed.
```

Comment:
"...expected to move out of the Incubation stage..."
If we are removing the Graduation stage, maybe we need to reword this sentence? or is the expectation that they will move to the Emeritus stage?

Reply:
I agree, good catch.

Reply:
Thanks Yash.  I have updated this PR to remove that text.